TheMovieDB/HorizontalScrollView.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  HorizontalScrollView.swift
+//  TheMovieDB
+//
+//  Created by Intern on 13/07/23.
+//
+
+import SwiftUI
+
+struct RectangleView: View {
+    @State var title: String
+    @Binding var currentPage: Int
+    var body: some View {
+        Button {
+            currentPage = Int(title)!
+        } label: {
+            ZStack {
+                Rectangle()
+                    .fill(Color.yellow)
+                    .frame(width: 50, height: 50, alignment: .center)
+                Text(title)
+                    .foregroundColor(.black)
+            }
+        }
+
+    }
+}
+struct HorizontalScrollView: View {
+    @Binding var currentPage: Int
+    var count: Int
+    var body: some View {
+        VStack {
+            Divider()
+            ScrollView(.horizontal) {
+                HStack(alignment: .center, spacing: 10) {
+                    ForEach(1..<count+1) {
+                        i in RectangleView(title: "\(i)", currentPage: $currentPage)
```

Comment:
`ForEach(1..<count+1) { index in
      RectangleView(title: "\(index)", currentPage: $currentPage)
 }
`

don't use `i` because other developers don't know what i mean, you can change to index or other name

Variable names should be self-describing